typedoc.js

```diff
@@ -1,13 +1,19 @@
 // @ts-check
 
 module.exports = {
-  mode: 'file',
+  mode: 'modules',
   ignoreCompilerErrors: false,
-  exclude: ['**/*+(__tests__|internal|lib|node_modules)/**/*'],
-  excludeExternals: true,
+  'external-modulemap': '.*packages/([^/]+)/src/.*',
+  exclude: ['**/*+(__tests__|internal|lib|node_modules|demos)/**/*'],
+  excludeExternals: false,
   excludeNotExported: true,
   excludePrivate: true,
   theme: 'minimal',
   gitRevision: 'master',
-  readme: 'README.md'
+  readme: 'README.md',
+  name: '@feature-hub',
+  out: 'packages/website/build/feature-hub/@feature-hub',
+  tsconfig: 'tsconfig.json',
+  'sourcefile-url-prefix':
+    'https://github.com/sinnerschrader/feature-hub/tree/master/packages/'
```

Comment:
should we maybe insert `git rev-parse HEAD` here instead of blindly using master?